packages/hash/frontend/src/blocks/page/style.module.css

```diff
@@ -63,3 +64,19 @@
   max-width: 1200px;
   overflow-x: auto;
 }
+
+.Block__InsertBlock {
+  width: 100%;
+  max-width: 1200px;
+  position: absolute;
+  height: 30px;
+  margin-left: 32px;
+}
+
+.Block__InsertBlock__Bottom {
+  top: 100%;
+}
+
+.Block__InsertBlock__Top {
+  bottom: 100%;
+}
```

Comment:
Nit: I'd prefer if we have these styles in InsertBlock. I think it makes sense to use this file for scenarios where we can't use MUI to style the element in question. It's not a blocking change though and if it seems difficult, then we can leave as is

Reply:
I see your point but these are related to how we position the `InsertBlock` component inside the block view, which I think should be abstracted from component itself, and be defined in the component that renders it. We only use this component here, but would we reuse it somewhere else we probably wouldn't want this styles there.